js/shims.d.ts

```diff
@@ -9,13 +9,6 @@ import * as _$ from 'jquery';
 // Globals from flarum/core
 import Application from './src/common/Application';
 
-/**
- * Helpers that flarum/core patches into Mithril
- */
-interface m extends Mithril.Static {
-  prop: typeof Stream;
-}
```

Comment:
I guess you should do `m: Mithril.Static` in the `declare global` selection further below then?

Reply:
Not sure if that works. I had a hard time getting `m` global to properly work... if `m: Mithril.Static` doesn't work we can keep `interface m extends Mithril.Static {}` just fine I think.